src/Microsoft.AspNetCore.Authentication.Facebook/FacebookExtensions.cs

```diff
@@ -16,6 +16,9 @@ public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder build
             => builder.AddFacebook(FacebookDefaults.AuthenticationScheme, configureOptions);
 
         public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder builder, string authenticationScheme, Action<FacebookOptions> configureOptions)
-            => builder.AddOAuth<FacebookOptions, FacebookHandler>(authenticationScheme, configureOptions);
+            => builder.AddFacebook(authenticationScheme, authenticationScheme, configureOptions);
```

Comment:
For the `UseFacebook()` overload you'll get scheme name Facebook and display name Facebook. As soon as you add multiple schemes you'd want unique display names, no? or are you talking about having just one but with a non-default scheme name?